.github/workflows/build.yml

```
@@ -0,0 +1,118 @@
+name: build
+on:
+  pull_request:
+    branches:
+      - 'master'
+      - 'release-[0-9]+.[0-9]+'
+    paths:
+      - '**.cmake'
+      - '**/CMakeLists.txt'
+      - '**/CMakePresets.json'
+      - 'cmake.*/**'
+      - '.github/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+
+env:
+  BIN_DIR: ${{ github.workspace }}/bin
+  INSTALL_PREFIX: ${{ github.workspace }}/nvim-install
+
+jobs:
+  macos-universal:
+    runs-on: macos-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: ./.github/scripts/install_deps.sh
+
+      - run: printf 'NVIM_BUILD_TYPE=Release\n' >> $GITHUB_ENV
+
+      - name: Build universal binary
+        run: ./.github/scripts/build_universal_macos.sh
+
+  old_cmake:
```

Using `-` instead of `_` is more consistent with other names
```suggestion
  old-cmake:
```